### PR TITLE
Landing and guest screens 

### PR DESCRIPTION
**Pull request by ariisrael**
Lots of stubs and presentation-only screens here. The landing screen is the only "complete" screen, everything else was added to complete the guest navigation flow (only visible on iOS with the skip button for app store approval reasons). I'd recommend reviewing by looking at the files changed since I didn't break it into commits until all the work was done. 

https://user-images.githubusercontent.com/16139810/114599284-e3593580-9c60-11eb-8b1c-cd188d977d34.mov

### CODE REVIEW

File: screens/LandingScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import { Platform, StyleSheet, SafeAreaView, View } from 'react-native'
+import Colors from '../constants/Colors'
+import { Logo, Button } from '../components'
+
+// TODO: Add type for navigation prop
+export default function LandingScreen({ navigation }): JSX.Element {
+    return (
+        <SafeAreaView style={styles.container}>
+            {Platform.OS === 'ios' && (
```
Comment (tabrindle) [inline screens/LandingScreen.tsx:10]: why is this iOS only?

File: screens/LandingScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        justifyContent: 'space-between',
+    },
+    buttonGroup: {
+        flex: 1.5,
+        alignSelf: 'stretch',
+    },
+    logoContainer: {
+        width: 200,
+        flex: 1,
+        marginTop: 100,
```
Comment (tabrindle) [inline screens/LandingScreen.tsx:56]: how does this look on different size screens?

File: components/Button.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+            >
+                {text}
+            </Text>
+        </TouchableOpacity>
+    )
+}
+
+const styles = StyleSheet.create({
+    button: {
+        paddingHorizontal: 20,
```
Comment (tabrindle) [inline components/Button.tsx:110]: now that you have a baseline of spacing, I would recommend abstracting this to a "themes" or "constants" file to keep consistency throughout the app. 

also, probably dont need paddingVertical and paddingHorizontal if they are the same value.

File: components/Guest.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import Button from './Button'
+import { useNavigation } from '@react-navigation/native'
+import { View, Text } from 'react-native'
+
+export default function Guest(): JSX.Element {
+    const navigation = useNavigation()
+    return (
+        <View style={{ alignSelf: 'stretch' }}>
+            <Text style={{ marginVertical: 20, textAlign: 'center' }}>
```
Comment (tabrindle) [inline components/Guest.tsx:10]: There's really nothing wrong with inline styles, especially for a single layout property but for consistency I would vote for using a style sheet.

File: screens/index.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import LandingScreen from './LandingScreen'
+import LogInScreen from './LogInScreen'
+import SignUpScreen from './SignUpScreen'
+import LearnScreen from './LearnScreen'
+import OrderScreen from './OrderScreen'
+import ShareScreen from './ShareScreen'
+import AccountScreen from './AccountScreen'
+
+export {
+    LandingScreen,
```
Comment (tabrindle) [inline screens/index.ts:10]: the pedantic side of me enjoys seeing these alphabetized as the list gets longer. Same with imports above. /shrug

**Review by tabrindle — APPROVED** [review]
A few very minor comments, nothing blocking. Looks like a good start!

File: screens/LandingScreen.tsx (hunk 1 above)
Comment (ariisrael) [reply screens/LandingScreen.tsx:10]: Only the iOS App Store reviewers require guest functionality, so product only wants to let users skip sign up or log in on iPhones. This is also the current functionality of the legacy apps!

**Pull request merged**